coding-concepts.js

```diff
@@ -9,46 +9,45 @@
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ["tangerine", "magenta", "lilac", "daffodil" , "indigo"]
+// b) Verify and explain: adds an aditional string named indigo into the array
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: correct ; console log is verifying the number of characters of the string, starting at 1. 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: o
+// b) Verify and explain: output: o ;  console log is indexing the variable greeting and finding the 4th character left to right counting from zero.
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: JavaScript
+// b) Verify and explain: output :  ; console log is indexing the variable index which has a value of 1 indexing the first string of the array. I believed that the index would return the first string of the array.
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
-
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: ["SATURDAY", "SUNDAY"]
+// b) Verify and explain: error, toUpperCase() is not a function. I believe toUpperCase() needs to be in a function, or possibly declared as a variable.
```

Comment:
.toUpperCase is a built-in method made for strings. This is an array of strings so the method will not act on it.